Gemfile

```diff
@@ -2,7 +2,8 @@ source 'https://rubygems.org'
 
 gem 'rails', '4.1.5'
 
-# gem 'activerecord-jdbc-adapter'
+gem 'activerecord-jdbc-adapter'
```

Comment:
Please change to `:github => 'josephbridgwaterrowe/activerecord-relativity-adapter'`